src/stores/login.ts

```diff
@@ -1,27 +0,0 @@
-import create from "zustand";
```

Comment:
Why delete the login store? This would have been a much nicer way of managing the global state than parsing JSON from local storage in every single component that needs the login state.

It's not worth making a blocker for this PR, but I think we should really consider using Zustand for our global state needs since the localstorage method isn't going to scale once we get into the schedule data and things like that.

Reply:
I decided to use it for login because it seems that when using the login store, a refresh of the page logs the user out. If you know of a better way, I'm happy to use login store again

Reply:
Oh I see. Yeah Zustand doesn't persist anything in local storage, its the same as Redux or MobX or Recoil or any other state library. So what you would do is store the login info in localstorage once and then use the Zustand store to access that globally throughout the application. This way you only parse the JSON from localstorage once when the application loads for the first time

Reply:
Ah I see! Okay, I think I'll change that back next sprint when I do some updates as some new functionality from the backend comes in